geomstats/_backend/tensorflow/__init__.py

```diff
@@ -748,6 +748,10 @@ def tril(m, k=0):
     return tf.linalg.band_part(m, -1, 0)
 
 
+def diag_indices(*args, **kwargs):
```

Comment:
Same comment here, can we get a full tensorflow implementation instead?

Reply:
I have removed it. I am not using it anyway. Regarding cleaning, there are tensorflow operations based on Numpy API but it is still experimental (https://www.tensorflow.org/api_docs/python/tf/experimental/numpy). If this is present we would definitely need to migrate to it.